src/dependencies.jl

```diff
@@ -106,7 +106,7 @@ end
 can_use(::Type{AptGet}) = has_apt && OS_NAME == :Linux
 package_available(p::AptGet) = can_use(AptGet) && !isempty(available_versions(p))
 function available_versions(p::AptGet)
-    vers = ASCIIString[]
+    vers = String[]
```

Comment:
Actually hold off on tagging Compat again, the way the String deprecation was done broke a lot of existing packages on 0.4 and should be fixed first.